src/components/backend-ai-storage-list.ts

```diff
@@ -834,10 +835,23 @@ export default class BackendAiStorageList extends BackendAIPage {
                 <img
                   id="filebrowser-img"
                   alt="File Browser"
-                  src="./resources/icons/filebrowser.svg"></img>
+                  src="./resources/icons/filebrowser.svg" />
                 <span>${_t('data.explorer.ExecuteFileBrowser')}</span>
             </mwc-button>
           </div>
+          <div id="">
```

Comment:
Is there any reason to add an empty id here?

Reply:
no. I will remove it!